src/commands/copy/__test__/copy.test.ts

```diff
@@ -151,4 +151,140 @@ describe('copyFiles', () => {
     assert.equal(tiffSource?.contentType, 'binary/octet-stream');
     assert.equal(tiffTarget?.contentType, 'binary/octet-stream');
   });
+
+  it('should not copy files when different size or multihash but no force', async () => {
+    await Promise.all([
+      fsa.write('memory://source/topographic.json', Buffer.from(JSON.stringify({ test: true })), {
+        contentType: 'application/octet-stream',
+        metadata: {
+          multihash: '12206fd977db9b2afe87a9ceee48432881299a6aaf83d935fbbe83007660287f9c2e',
+          unique: 'fileA',
+        },
+      }),
+      fsa.write('memory://target/topographic.json', Buffer.from(JSON.stringify({ test: true })), {
+        contentType: 'application/octet-stream',
+        metadata: { multihash: 'different', unique: 'fileB' },
```

Comment:
We should probably use a valid multihash. I would expect that the invalid multihash could trigger a condition which cancels the upload for a different reason than the one stated in the test description.

Reply:
nixpkgs uses a ["fake" hash](https://github.com/NixOS/nixpkgs/blob/cfbeda848e75c62d6facd3ae40d6f239dcee9375/lib/deprecated/misc.nix#L328-L330) which is visually distinct from any hash you'd actually encounter. We could do the same with something like "1220ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff".